trp_project_scrum/security/ir_rule.xml

```diff
@@ -0,0 +1,18 @@
+<odoo>
+
```

Comment:
we need rules for all models that are not restricted already by some other inherited group